### ISSUE TITLE
reboot the project ?

### ISSUE THREAD

**Issue by MasseGuillaume** [timowest/scalagen#85]
/cc @timowest @nightscape

hello. I'm currently working on Scala-Native and we have a lot of java code to port from java to scala. I would like to spend some time rebooting this project if there is an interest.

**Comment by nightscape** [comment]
Hi @MasseGuillaume,

that would be quite nice. My branch contains a huge refactoring that should make the code more maintainable and also has Java 8 and comment support. What I think would be really helpful is a SBT plugin in addition to the Maven one. I don't know if @timowest is still reading the issues here, so maybe a fork to the scalacenter organization (of course granting credit where credit is due :wink:) and adding some more collaborators could get this project moving again.

**Comment by philwalk** [comment]
I would be interested in this as well

**Comment by nightscape** [comment]
Hey @timowest,

as you don't seem to have much time for this project, how about creating a scalagen organization and moving it over there and granting a few people admin rights?

Best
  Martin

**Comment by joshlemer** [comment]
@nightscape Did you ever create a scalagen organization to migrate this project?

**Comment by nightscape** [comment]
Nope, not yet. I was hoping for a response from @timowest, because it would be much easier (in terms of making search engines and people find the new location) to transfer the repo to a new organization.
But maybe it's a good time to just move ahead.

**Comment by nightscape** [comment]
Ok, so here we go: https://github.com/scalagen/scalagen
Let the merge-party begin :wink: 
Anybody who is willing to make contribution, do code reviews, tell PR authors to redirect their PRs to the new repo, etc. please get in touch with me :smiley:
